### ISSUE TITLE
sample-framebuffer yields a sokol assert failure

### ISSUE THREAD

**Issue by Larpon** [edubart/sokol_gp#5]
Hi, thanks for making this nice project available!

I've cloned the repository, ran `make`, and could successfully run all `./build/sample-*` executables *except* the `sample-framebuffer` which outputs:
```
./build/sample-framebuffer 
sg_apply_pipeline: pipeline depth pixel_format doesn't match pass depth attachment pixel format
^^^^  SOKOL-GFX VALIDATION FAILED, TERMINATING ^^^^
sample-framebuffer: thirdparty/sokol_gfx.h:13735: _sg_validate_end: Assertion `0' failed.
Aborted (core dumped)
```

**Comment by edubart** [comment]
You need to provide more information on your system and backend. Also check https://github.com/edubart/sokol_gp/issues/3
seems like it is related.

**Comment by edubart** [comment]
I've noticed that my fix for Metal for issue #3 broke the framebuffer sample and I have fixed in the latest commit. This issue was just in the sample, not in the library. Sokol app still need a way to disable depth stencil for all platforms though, so we can skip the overhead of creating the unused depth stencil image.

Thanks for reporting and letting me know that the framebuffer sample was broken.

**Comment by Larpon** [comment]
Thanks for quick fix!

Sorry for the missing system info.

For the record:
Linux, Kubuntu 20.04
Thinkpad T450s
Intel i7 Core, Intel 5500 integrated graphics
12GB of ram
